public/style_editor.js

```diff
@@ -1333,13 +1339,11 @@ function encstr(s) {
 }
 
 function mkbutton2(name, val) {
-  return "<input type=button class=btn onclick='SetToAndFormat(\""+val+"\")' value='"+enc(name)+"'>\n";
-  //  return "<span class=btn onclick='SetTo(\""+name+"\")'>"+enc(name)+"</span>\n";
+  return "<input type=button class='btn' onclick='SetToAndFormat(\""+val+"\", event)' value='"+enc(name)+"'>\n";
```

Comment:
This is just an another anomaly?
The master does not have that removed comment line in it.
The return line is identical.
Must be GitHub?
https://github.com/profezzorn/ProffieOS-StyleEditor/blob/417f189bc7b0e88fd0b9e3e38fe69ed84078088e/public/style_editor.js#L1328C3-L1328C113